src/types.ts

```diff
@@ -57,16 +57,16 @@ export type NamedTensorMap = {
  * converted to a ConfigDict that has renamed fields (TS naming conventions)
  * and support for Enums.
  */
-export type JsonValue = boolean|number|string|null|JsonArray|JsonDict;
-export interface JsonDict {
-  [key: string]: JsonValue;
+export type PyJsonValue = boolean|number|string|null|PyJsonArray|PyJsonDict;
```

Comment:
Can you add a sentence or two to the doc string of this type to describe why it's called `PyJsonValue` (i.e., that it contains keys in Python Keras-compatible format, that is, snake_case)